nvflare/app_opt/pt/file_model_locator.py

```diff
@@ -47,6 +47,7 @@ def _initialize(self, fl_ctx: FLContext):
             raise ValueError(
                 f"pt_persistor_id component must be PTFileModelPersistor. " f"But got: {type(self.model_persistor)}"
             )
+        self.model_persistor.load_model(fl_ctx)
```

Comment:
Shouldn't be model locator's job to call load_model of the persistor here.

Reply:
The purpose of this call to enforce the persistence_manager creation in the PTFileModelPersistor. Changed it to use lazy initialization there.